sdk/cosmos/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/core/generator/AbstractQueryGenerator.java

```diff
@@ -28,6 +28,8 @@
  */
 public abstract class AbstractQueryGenerator {
 
+    private static String tableName = "r";
```

Comment:
If you plan to update `tableName` in the code below, then I would suggest keeping this as a separate constant value. Something, like this - 

```suggestion
    private static final String CONSTANT_TABLE_NAME = "r";
```

Reply:
Done.